app/views/docs/certificates.phtml

```diff
@@ -9,12 +9,34 @@
 
 <h2>Debugging</h2>
 
-<p>In case you&#39;re still struggling with getting your certificate to work, check the Appwrite certificates worker log. You can do that with the following command:</p>
+<p>In case you're still struggling with getting your certificate to work, check the Appwrite certificates worker log. You can do that with the following command:</p>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
     <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose logs appwrite-worker-certificates</code></pre>
 </div>
 
+<h2>Generation cycle</h2>
+
+<p>Appwrite auto-generates certificate for your main domain when you first visit it. If your browser shows an insecure connection warning, you must proceed to trigger certificate generation. Domain in environment variable _APP_DOMAIN is considered your main domain. If you didn't set this variable, the first domain you visit would be marked as the main domain for your Appwrite instance. Appwrite follows this concept of the main domain to prevent generating certificates for domains you don't allow. Keep in mind that you can add a domain as a custom domain in your project settings to enable certificate generation for a any domain.</p>
```

Comment:
either > auto-generates a certificate
or > auto-generates certificates

You're missing an article here (a, an, the, etc.)

Also here => **The domain** in environment variable _APP_DOMAIN is considered your main domain.

Reply:
`Appwrite auto-generates a certificate` applied. When you first visit, it only generates one.

`The domain in ...` applied.